<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>juego</class>
 <widget class="QMainWindow" name="juego">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1051</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="vida1">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="imagenes.qrc">:/vidas.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="vida2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="imagenes.qrc">:/vidas.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="vida3">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="imagenes.qrc">:/vidas.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Puntaje</string>
    </property>
   </widget>
   <widget class="QLabel" name="punt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gotas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>1031</width>
      <height>551</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Numero Bombas</string>
    </property>
   </widget>
   <widget class="QLabel" name="N_Bombas">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar_Juego"/>
    <addaction name="actionCargar_Juego"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuardar_Juego">
   <property name="text">
    <string>Guardar Juego</string>
   </property>
  </action>
  <action name="actionCargar_Juego">
   <property name="text">
    <string>Cargar Juego</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
